src/components/header.tsx

```diff
@@ -60,7 +65,11 @@ export const Header: React.FC<Props> = ({
   <>
     <LogoContainer>
       <Link to="/">
-        <Logo />
+        {overrides.images.logo ? (
+          <img alt="logo" src={overrides.images.logo} />
+        ) : (
+          <Logo />
```

Comment:
maybe this override should be in the `Logo` component?

Reply:
The logo component is actually just an svg. Ideally I'm thinking we have a central images index that the overrides get applied over, so the majority of the code has no clue about it.

Looking into it, I think my fear of not using svg in img tag is unfounded. IE has support now since version 9:

https://caniuse.com/#feat=svg-img

Will switch it up